website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,40 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checkout, showError) {
+    var ret = $.Deferred();
```

Comment:
This doesn't seem to be every resolved anywhere (i.e. I don't see any calls to `ret.resolve()` or `ret.fail()`). Any reason not to just return the `jqxhr` (the return value of `$osf.ajaxJSON`?